module_1/first-react-app/src/components/Movie.js

```diff
@@ -0,0 +1,32 @@
+import { UNKNOWN_MOVIE } from '../literals';
+import { NOT_APPLICABLE } from '../literals';
+import '../css/Movie.css';
+
+const Movie = ({ imageUrl, movieName, releaseYear, genres, onTileClick }) => {
+
+    const displayedGenre = genres ? genres.flat().join(", ") : NOT_APPLICABLE;
+    const displayedMovieName = movieName ? movieName : UNKNOWN_MOVIE;
+    const displayedReleaseYear = releaseYear ? releaseYear : NOT_APPLICABLE;
+
+    return (
+        <>
+            <div className="row" >
+                <div className="col s12 movie-list">
+                    <div className="card large" onClick={onTileClick} >
+                        <div className="card-image movie-tile" >
+                            <img src={imageUrl} height='100%' alt='...' />
```

Comment:
please put the correct value for the `alt`

Reply:
Have changed it to be set same as the movie name!